source/new.rst

```diff
@@ -696,6 +686,80 @@ Notes on the verified group protocol
   Carol could send the key to further verified groups
   that Bob is not part of.
 
+Dealing with key loss and compromise
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If a user looses their device
+they can setup a new device
+and regain access to their inbox.
+However they may loose their secret key.
+
+They can generate a new key pair.
+Autocrypt will distribute their new public key
+in the Autocrypt headers
+and opportunistic encryption will switch to it automatically.
+
+Verified groups will remain unreadable
+until the user verifies a contact from that group.
+Then the contact can update the key used in the group.
+This happens by sending a "vg-member-added" message
+to the group.
```

Comment:
in reality it's more a "member-changed" message i guess because the member already exists but its key is replaced.

Reply:
I changed the name from member-added to member-setup. We are not talking about member-changed messages anywhere else. So we would need a section introducing them and describing them.
I think member-setup messages work for both.. introducing new users and updating existing ones. So introducing a new message type felt unnecessary to me.